rllib/policy/policy_map.py

```diff
@@ -141,7 +140,7 @@ def create_policy(
             self.session_creator,
             self.seed,
         )
-        self.insert_policy(policy_id, policy, config_override)
```

Comment:
This was actually a bug: Policy got created w/o(!) its specific config override, but then added anyways and the override was only recorded in the PolicySpec (for the state).